testdata/go/v3/memcached-operator/api/v1alpha1/memcached_types.go

```diff
@@ -29,21 +29,32 @@ type MemcachedSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// Size defines the number of Memcached instances
+	// The following markers will use OpenAPI v3 schema to validate the value
+	// More info: https://book.kubebuilder.io/reference/markers/crd-validation.html
+	// +kubebuilder:validation:Minimum=1
+	// +kubebuilder:validation:Maximum=3
+	// +kubebuilder:validation:ExclusiveMaximum=false
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
```

Comment:
```suggestion

         // Size defines the number of Memcached instances
	// +operator-sdk:csv:customresourcedefinitions:type=spec
```

@NikhilSharmaWe we need the space and the description on top of the marker